solara/website/pages/documentation/examples/utilities/todo.py

```diff
@@ -151,25 +106,23 @@ def create_new_item(*ignore_args):
 
```

Comment:
If this is an example of how to manage state, it would be even more valuable to show _how_ the todos can be reused. Now the Page component is effectively a Todo component, but it's not reusable as State can only manage a single list of Todos.